### ISSUE TITLE
Whether there are more duplicate data in the test set and training set？

### ISSUE THREAD

**Issue by zxzhang8** [xing-hu/EMSE-DeepCom#28]
Hello, thank you for sharing the code and data set.
But I found 4463 pieces of data in the test set appeared in the training set. Was it deliberately designed like this?
